static/templates/project/submission-review-tabular.html

```diff
@@ -10,13 +10,23 @@
                   <md-tooltip>{{ review.resolvedData.name }}</md-tooltip>
                 </span>
         </div>
-        <div class="_project-rating-header">
+        <div class="_project-rating-header layout-row layout-align-start-center">
             <div>
                 <div class="_large-header">Give priority access to great workers</div>
                 <div class="_secondary-text-small">Priority workers see your tasks before anyone else. The more you
                     rate, the better your results get.
                 </div>
             </div>
+            <div flex>
+              &nbsp
+            </div>
+
+            <div class="light placeholder">
+              <!-- Boomerang Status -->
+              <div class="breadcrumb">
+                <a class="current">Your Priority Workers</a><a ng-class="{currentnormal: review.resolvedData.min_rating <= 2}">Your Normal Workers</a><a ng-class="{currentlate: (review.resolvedData.min_rating < 1.99)}">Your Late Workers</a>
```

Comment:
no need to start from scratch, just change the appropriate tags and css rules and you should be fine